go.mod

```diff
@@ -1,37 +1,58 @@
 module github.com/kubernetes-csi/csi-proxy
 
-go 1.23
```

Comment:
This project became incompatible with go1.23 and above because of https://github.com/kubernetes-csi/csi-proxy/issues/361. Please rebase.